config/locales/en_surveys.yml

```diff
@@ -0,0 +1,26 @@
+---
+en:
+  activemodel:
+    attributes:
+      survey_answer:
+        choices: Choices
+    errors:
+      models:
+        survey_answer:
+          attributes:
+            choices:
+              missing: are missing
+  # Remove when merge: https://github.com/decidim/decidim/pull/11450/files
+  decidim:
+    components:
+      surveys:
+        settings:
+          global:
+            send_confirmation_email: Send an email with the answers when the survey is answered
+    surveys:
+      survey_confirmation_mailer:
+        confirmation:
+          body: You have answered the questionnaire %{questionnaire_title} correctly in %{participatory_space}
+          subject: Confirmation of response to the questionnaire %{questionnaire_title}
+        export_name: Survey answers
+    # Remove when merge: https://github.com/decidim/decidim/pull/11450/files
```

Comment:
This PR has already been merged. Please remove this block from all languages

Reply:
Removed. I see inconsistency in these files _surveys.yml between languages, Spanish and English version have only 13 lines while Occitan and Catalan versions contain additional blocks. What should we do with that additional blocks (see ca_surveys.yml and oc_surveys.yml)?

Reply:
I was wrong :pray:  , we already have it checked to do during the next upgrade: `spec/features/overrides_and_customizations_spec.rb`. I've already re-added them.